src/components/Avatar/Styles/rtl.scss

```diff
@@ -6,3 +6,12 @@
     }
   }
 }
+
+.avatar-group-spaced {
+  &-rtl {
```

Comment:
Once moved as `&-spaced {...}`, in `group.scss` there will be no need to create a separate `&-rtl` it may use the above block as an override in the cascade.